test/fixtures/linter/passing.sql

```diff
@@ -1 +1 @@
-select a from b
\ No newline at end of file
+select a from b
```

Comment:
Given the filename, I assume this file was meant to pass linting, but it was missing a trailing newline, which caused it to fail. See `test/cli/commands_test.py::TestProgressBars` for some of the affected tests.